src/c_api/eigensolver/gen_eigensolver.cpp

```diff
@@ -81,4 +115,34 @@ void dlaf_pzhegvx(const char uplo, const int m, dlaf_complex_z* a, const int ia,
   pxhegvx<std::complex<double>>(uplo, m, a, ia, ja, desca, b, ib, jb, descb, w, z, iz, jz, descz, *info);
 }
 
+void dlaf_pssygvx_factorized(const char uplo, const int m, float* a, const int ia, const int ja,
```

Comment:
I would use the correct name from the beginning
```suggestion
void dlaf_pssygvd_factorized(const char uplo, const int m, float* a, const int ia, const int ja,
```